### DISCUSSION TITLE
RPI-RF-MOD mit "Standard" FTDI Adapter

### DISCUSSION THREAD

**Discussion by MaximilianMM** [OpenCCU/OpenCCU#1424] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,
ich würde gerne RaspberryMatic auf meinem Proxmox-Server installieren. Dazu würde ich das „RPI-RF-MOD“ Modul benutzen. Nun stelle ich mir die Frage, ob als Alternative für die HB-RF-USB-Platine auch ein „Standard USB zu TTL Adapter“ verwendet werden kann. Diese benutzen ja eigentlich auch immer den „FT232RL“ IC. Mit einigen Jumper-Kabeln sollte man das doch zum Laufen bringen, oder?
LG Max

**Comment by jens-maus** [comment, marked as answer]
Nein, das wird nicht so einfach funktionieren, denn für den kombinierten homematic/homematicIP Betrieb ist es notwendig, das garantierte latenzzeiten eingehalten werden. Und das erbringt momentan nur eine HB-RF-USB bzw. HB-RF-ETH Platine in Kombination mit dem passenden kernel modul (generic_raw_uart). Und dieses Kernel modul setzt den Einsatz eines HB-RF-USB/HB-RF-ETH voraus, da nur damit die korrekt Betrieb gewährleistet wird.
